docsteady/templates/dm-tpr.latex.jinja2

```diff
@@ -205,9 +209,15 @@ Primavera milestones related to the test campaign.
 {% if testplan.pmcs_activity %}
 {{ testplan.pmcs_activity }}
 {% else %}
+  {% if testplan.milestone_id == "" %}
```

Comment:
The previous line should now  probably be  "Primavera milestones related to the test campaign:"  plus there is an extra blank line there it looks like